pycbc/pnutils.py

```diff
@@ -545,14 +549,22 @@ class Params:
                                  n_points)
         track_t = numpy.array([findchirp_chirptime(float(mass1), float(mass2), 
                                         float(f), pn_2order) for f in track_f])
-    elif approximant == 'SEOBNRv2_ROM_DoubleSpin':
+    elif approximant in ['SEOBNRv2', 'SEOBNRv2_ROM_DoubleSpin', 'SEOBNRv2_ROM_DoubleSpin_HI']:
         f_high = get_final_freq('SEOBNRv2', mass1, mass2, spin1, spin2)
         track_f = numpy.logspace(numpy.log10(f_low), numpy.log10(f_high),
                                  n_points)
         track_t = numpy.array([
                 lalsimulation.SimIMRSEOBNRv2ROMDoubleSpinTimeOfFrequency(f,
                     solar_mass_to_kg(mass1), solar_mass_to_kg(mass2),
                     float(spin1), float(spin2)) for f in track_f])
+    elif approximant in ['SEOBNRv4', 'SEOBNRv4_ROM']:
+        f_high = get_final_freq('SEOBNRv4', mass1, mass2, spin1, spin2)
+        track_f = numpy.logspace(numpy.log10(f_low), numpy.log10(f_high),
+                                 n_points)
+        track_t = numpy.array([
+                lalsimulation.SimIMRSEOBNRv4ROMDoubleSpinTimeOfFrequency(f,
+                    solar_mass_to_kg(mass1), solar_mass_to_kg(mass2),
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->